docker-compose/run-dev.sh

```diff
@@ -1,25 +1,7 @@
 #!/bin/sh
 
-kuzzle=${KUZZLE_HOST:-kuzzle:7511}
```

Comment:
I don't know if this was useful anymore, but I'm sure it didn't work due to a config bug I discovered this weekend (we make a wrong use of the `webpack.DefinePlugin`). Anyway, if this happens to be useless, let's take it out.